Formula/proj.rb

```diff
@@ -17,6 +17,7 @@ class Proj < Formula
     depends_on "libtool" => :build
   end
 
+  depends_on "libtiff" => :build
```

Comment:
Judging by the linkage errors, this looks like it's a runtime dependency - not just a build one.

Reply:
Hmm. Removing the "build" directive doesn't seem to have worked.